enrichissement.py

```diff
@@ -125,6 +128,46 @@ def manage_column_final(df):
                              'geolocCommuneEtablissement'])
     return df
 
+  
+
+
+#Enrichir avec le departement des acheteur et des etablissements. Base departement francais.
+def extraction_code_postal(code_postal):
+    """renvoie le code postal en prenant en compte les Drom
+    code_postal est un str"""
+    try:
+        code = code_postal[:2]
+        if code == "97" or code == "98":
+            code = code_postal[:3]
+        return code
+    except:
```

Comment:
On a eu besoin d'une méthode similaire dans planR, elle est quasi identique à un détail près :
```suggestion
    except IndexError:
```
Plus explicite et permet d'éviter de s'arracher les cheveux si une autre erreur est introduite par effet de bord (par example une méthode qui change le type de `code_postal` par erreur).